src/app/backup/details/snapshot/template.html

```diff
@@ -50,6 +50,13 @@
         <div value>{{backup.spec?.clusterId}}</div>
       </km-property>
     </mat-card-content>
+
+    <mat-card-content fxLayout="row">
```

Comment:
Is `fxLayout="row"` needed? There is only 1 element inside it.

Reply:
Fixed - including other remarks.